vlm/src/main/scala/geotrellis/contrib/vlm/RasterSource.scala

```diff
@@ -46,8 +47,9 @@ trait RasterSource extends CellGrid with AutoCloseable with Serializable {
     def crs: CRS
     def bandCount: Int
     def cellType: CellType
-    def cellSize = rasterExtent.cellSize
+    def cellSize: CellSize = rasterExtent.cellSize
     def rasterExtent: RasterExtent
+    def overviewsRasterExtents: List[RasterExtent]
```

Comment:
`overviewsRasterExtents` seems too specific. What about GeoTrellis indexed layers?

The other problem is that they can easily blow out the `Int` cell count of `RasterExtent`.
The two options remaining are:
- `CellSize`: assume that extents are always the same for levels of detail
- `List[GridExtent]`: wide enough and convertible to `RasterExtent`